pages/advanced/dependent-devices.md

```diff
@@ -0,0 +1,229 @@
+* A method of reliably powering the Raspberry Pi 3 and the micro:bit.
+* Docker installed on your local machine.
+* A resin.io account.
+
+## Getting Help
+
+Before we get started building something cool, lets just point out some places to get help.
+{{import "usingSupport"}}
+
+To help us understand all the moving parts in resin.io dependent devices support, lets first define a few terms that will be used later in the guide.
+* **Gateway Application:**
+> The gateway application is responsible for detecting, provisioning and managing dependent devices belonging to one of its dependent applications. This is possible leveraging a new set of [endpoints](https://github.com/resin-io/resin-supervisor/blob/master/docs/dependent-apps.md) exposed by the Resin Supervisor. The [edge-node-manager][edge-node-manager-link] is an example of a gateway application.
+
+* **Gateway Device:**
+> The gateway device runs the gateway application and has the needed on-board radios to communicate with the managed dependent devices, for example: Bluetooth, WiFi, LoRa or ZigBee.
+
+* **Dependent Application:**
+> A dependent application is a Resin application that targets devices not capable of interacting directly with the Resin API and is scoped under the gateway application. A dependent application follows the same development cycle as a conventional Resin application with a few key differences: it does not support Dockerfile templating, the Dockerfile must target an x86 base image and the actual firmware must be stored in the /assets folder within the built docker image.
+
+* **Dependent Device:**
+> A dependent device is a device not capable of interacting directly with the Resin API - the reasons can be several, the most common are: no direct Internet capabilities and not able to run the Resin OS (being a microcontroller, for example)
+
+
+## Let's Jump In
+
+If you don't already have a resin.io account head over to our [signup page][link-to-signup], during the sign up process you will be asked to set up an SSH key so you can securely push code.
+{{import "sshKey/add"}}
+
+## Creating an Application
+To create an application simply type in a name, select the Raspberry Pi 3 device type from the drop down list and click the create button. You should now be taken to the dashboard of your newly created application:
+
+<img src="/img/dependent_devices_dashboard/select_fleet_type.jpg" width="80%">
+
+This dashboard may not look like much right now, but this is where you will command and manage a whole fleet of [micro:bits][micro-bit-link].
+
+## Enabling dependent device support
+To enable dependent device support we need to set two fleet wide configuration variables, defining `RESIN_SUPERVISOR_DELTA` as `1` and `RESIN_UI_ENABLE_DEPENDENT_APPLICATIONS` as `1`
+
+![Setting the fleet configuration to enable dependent device support](/img/dependent_devices_dashboard/EnableSupport.jpg)
+
+## Adding Your First Device
```

Comment:
I found this section a bit confusing because, since it uses the generic resin docs, it doesn't refer to the device and application as the *gateway* device and the *gateway* application. I think we should use consistent language throughout the dependent device documentation and clearly refer to gateway vs. dependent devices and applications wherever we can.